[Reland] Add sym_size/stride/numel/storage_offset to native_function.yaml

SherlockNoMad opened the pull request:
Differential Revision: D46459100

angelayi reviewed (approved):
stamping based on previous land attempts

SherlockNoMad committed:
[Reland] Add sym_size/stride/numel/storage_offset to native_function.yaml (#103107)

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/103107

Reviewed By: angelayi

Differential Revision: D46459100

Pulled By: SherlockNoMad

fbshipit-source-id: f7c65880e77b1d78c426e0e77b871fbe6583976f

SherlockNoMad commented:
@pytorchbot merge

soulitzer reviewed (approved):
stamping for reland
